README.md

```diff
@@ -1 +1,415 @@
-# project-cezary
\ No newline at end of file
+# Soil moisture system using grove capicty moisture sensor, Wio E5 dev board and Wio E5 mini board
+## Introduction
+![System schema](./docs/asset/project_cezary_1.drawio.png)
+Soil moisture system includes:
+* Reading soil mositure level via ADC on mini board.
+* Sending data about mositure level via Lora to dev board.
+* Dev board recive this data and send order to mini board about, what color rgb led, should lights.
+## Technologies
+Project is created with:
+* Zephyr: 3.5.0
+* West: 1.2.0
+## Preparation
+1. It is necessary to have Zephyr already installed to build or flash application.
+   * Go to the official Zephyr RTOS website (https://www.zephyrproject.org/) to download the latest version.
+   * Follow the installation instructions on the website to install Zephyr RTOS on your system.
+2. Install Build Tools:
+* Zephyr RTOS uses build tools like CMake and ninja. You have to follow this commmands to install:
+* CMake:
+```
+sudo apt-get update
+sudo apt-get install cmake
+```
+* Ninja:
+```
+sudo apt-get update
```

Comment:
As this is one preparation process, you don't have to add `sudo apt-get update` for each build tool. I would leave the first one only.